spec/support/helpers/api_helper.rb

```diff
@@ -44,6 +44,19 @@ def read_error_response
     parse_json_file(ERROR_API_RESPONSE)
   end
 
+  def build_stubbed_order
```

Comment:
you're not stubbing anything here. the naming is misleading. you should consider moving this into the factories